### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by natemcmaster**
Manually updating to latest core-setup versions

### CODE REVIEW

File: eng/Versions.props [hunk 1]
```diff
@@ -19,7 +19,7 @@
   -->
   <PropertyGroup Label="Automated">
     <!-- Packages from dotnet/core-setup -->
-    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview3-27419-3</MicrosoftNETCoreAppPackageVersion>
+    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview3-27420-6</MicrosoftNETCoreAppPackageVersion>
     <!-- Packages from dotnet/corefx -->
     <MicrosoftWin32RegistryPackageVersion>4.6.0-preview3.19115.9</MicrosoftWin32RegistryPackageVersion>
```
Comment (natemcmaster) [inline eng/Versions.props]: Latest corefx changes in release/3.0 of core-setup is 4.6.0-preview3.19120.17, but I don't see new builds of core-setup with these changes.

cc @dagood

Comment (dagood) [reply eng/Versions.props]: Core-Setup release/3.0 builds are failing during blob feed publish. Kicking off a new one and taking a look at the failure.

Comment (dagood) [reply eng/Versions.props]: Filed https://github.com/dotnet/arcade/issues/2086 for the failure.

New build is here, to track progress (or another failure): https://dev.azure.com/dnceng/internal/_build/results?buildId=102814.

Comment (natemcmaster) [reply eng/Versions.props]: Success! Updated this PR.

**Pull request merged**